src/function/table/table_scan.cpp

```diff
@@ -421,47 +419,27 @@ static unique_ptr<FunctionData> TableScanDeserialize(Deserializer &deserializer,
 		throw SerializationException("Cant find table for %s.%s", schema, table);
 	}
 	auto result = make_uniq<TableScanBindData>(catalog_entry.Cast<DuckTableEntry>());
-	deserializer.ReadProperty(103, "is_index_scan", result->is_index_scan);
+	deserializer.ReadDeletedProperty<bool>(103, "is_index_scan");
 	deserializer.ReadProperty(104, "is_create_index", result->is_create_index);
-	deserializer.ReadProperty(105, "result_ids", result->row_ids);
+	deserializer.ReadDeletedProperty<unsafe_vector<row_t>>(105, "result_ids");
 	return std::move(result);
 }
 
-TableFunction TableScanFunction::GetIndexScanFunction() {
-	TableFunction scan_function("index_scan", {}, IndexScanFunction);
-	scan_function.init_local = nullptr;
-	scan_function.init_global = IndexScanInitGlobal;
-	scan_function.statistics = TableScanStatistics;
-	scan_function.dependency = TableScanDependency;
-	scan_function.cardinality = TableScanCardinality;
-	scan_function.pushdown_complex_filter = nullptr;
-	scan_function.to_string = TableScanToString;
-	scan_function.table_scan_progress = nullptr;
-	scan_function.get_partition_data = nullptr;
-	scan_function.projection_pushdown = true;
-	scan_function.filter_pushdown = false;
-	scan_function.get_bind_info = TableScanGetBindInfo;
-	scan_function.serialize = TableScanSerialize;
-	scan_function.deserialize = TableScanDeserialize;
-	scan_function.get_partition_stats = nullptr;
-	return scan_function;
-}
-
 TableFunction TableScanFunction::GetFunction() {
```

Comment:
This does not precisely match the previous index scan function, mainly causing issues with the partitioned scan functionality.